acp/identity/identity.go

```diff
@@ -112,7 +111,14 @@ func didFromPublicKey(publicKey *secp256k1.PublicKey, producer didProducer) (str
 
 // IntoRawIdentity converts an `Identity` into a `RawIdentity`.
 func (identity Identity) IntoRawIdentity() RawIdentity {
-	return newRawIdentity(identity.PrivateKey, identity.PublicKey, identity.DID)
+	privKeyBytes, _ := identity.PrivateKey.Raw()
+	pubKeyBytes, _ := identity.PublicKey.Raw()
```

Comment:
todo: Please return the errors, or at least panic if they are not nil.  Please do not ignore them unless you have a very well documented reason to do so.